_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,428 @@
+        unzip \
+        libcurl4-openssl-dev \
+        autoconf \
+        automake \
+        libtool \
+        cmake \
+        python3 \
+        libkrb5-dev \
+        curl
+
+# Copy function code
+WORKDIR ${FUNCTION_DIR}
+RUN curl -LJO https://artifacts.opensearch.org/reporting-cli/opensearch-reporting-cli-1.0.0.tgz
+RUN tar -xzf opensearch-reporting-cli-1.0.0.tgz
+RUN mv package/* .
+RUN npm install && npm install aws-lambda-ric
+
+# Build Stage 2: Copy Build Stage 1 files in to Stage 2. Install chromium dependencies and chromium.
+FROM node:lts-slim
+# Include global arg in this stage of the build
+ARG FUNCTION_DIR
+# Set working directory to function root directory
+WORKDIR ${FUNCTION_DIR}
+# Copy in the build image dependencies
+COPY --from=build-image ${FUNCTION_DIR} ${FUNCTION_DIR}
+
+# Install latest chrome dev package and fonts to support major char sets (Chinese, Japanese, Arabic, Hebrew, Thai and a few others)
+# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer installs, work.
+RUN apt-get update \
+    && apt-get install -y wget gnupg \
+    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
+    && apt-get update \
+    && apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 \
+      --no-install-recommends \
+    && apt-get remove -y google-chrome-stable \
+    && rm -rf /var/lib/apt/lists/*
+
+ENTRYPOINT ["/usr/local/bin/npx", "aws-lambda-ric"]
+
+ENV HOME="/tmp"
+CMD [ "/function/src/index.handler" ]
+
+```
+
+Next, run the following build command within the same directory that contains the Dockerfile:
+
+```
+docker build -t opensearch-reporting-cli .
+```
+
+### Step 2: Create a private repository with Amazon ECR
+
+Follow the instructions in the Amazon ECR user guide to create a repository with the name `opensearch-reporting-cli`. For instructions, see [Getting started with Amazon ECR using the AWS Management Console](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html).
+
+In addition to the Amazon EC2 instructions, you need to make the following adjustments for the Reporting CLI to function properly:
+
+Navigate to the test function **Configuration** tab to modify two required settings.
+- Make sure to set **Timeout** to at least 5 minutes to allow the reporting CLI to generate the report.
+- Change the setting for **Ephemeral storage** to 1024MB. The default setting is not a sufficient storage amount.
+{: .note }
+
+### Step 3: Run the push commands
+
+You need to get several commands from the AWS EC2 Console to run within the Dockerfile directory, as shown in the following image:
+
+![Amazon ECR console view push commands]({{site.url}}{{site.baseurl}}/images/dashboards/push-commands.png)
```

Comment:
somehow the image is very blurry on my side

Reply:
ok maybe i can get a clean one from Rupal. thanks